homeassistant/components/google_air_quality/sensor.py

```diff
@@ -0,0 +1,186 @@
+            "moderate_air_quality",
+            "poor_air_quality",
+        ],
+        value_fn=lambda x: x.indexes[0].category,
+    ),
+    AirQualitySensorEntityDescription(
+        key="local_aqi",
+        translation_key="local_aqi",
+        exists_fn=lambda x: x.indexes[1].aqi is not None,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.AQI,
+        value_fn=lambda x: x.indexes[1].aqi,
+    ),
+    AirQualitySensorEntityDescription(
+        key="local_category",
+        translation_key="local_category",
+        device_class=SensorDeviceClass.ENUM,
+        value_fn=lambda x: x.indexes[1].category,
+    ),
+    AirQualitySensorEntityDescription(
+        key="uaqi_dominant_pollutant",
+        translation_key="uaqi_dominant_pollutant",
+        device_class=SensorDeviceClass.ENUM,
+        value_fn=lambda x: x.indexes[0].dominant_pollutant,
+    ),
+    AirQualitySensorEntityDescription(
+        key="local_dominant_pollutant",
+        translation_key="local_dominant_pollutant",
+        device_class=SensorDeviceClass.ENUM,
+        value_fn=lambda x: x.indexes[1].dominant_pollutant,
+    ),
+    AirQualitySensorEntityDescription(
+        key="co",
+        translation_key="carbon_monoxide",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
+        value_fn=lambda x: x.pollutants.co.concentration.value,
+    ),
+    AirQualitySensorEntityDescription(
+        key="no2",
+        translation_key="nitrogen_dioxide",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
+        value_fn=lambda x: x.pollutants.no2.concentration.value,
+    ),
+    AirQualitySensorEntityDescription(
+        key="o3",
+        translation_key="ozone",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
+        value_fn=lambda x: x.pollutants.o3.concentration.value,
+    ),
+    AirQualitySensorEntityDescription(
+        key="pm10",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.PM10,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        value_fn=lambda x: x.pollutants.pm10.concentration.value,
+    ),
+    AirQualitySensorEntityDescription(
+        key="pm25",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.PM25,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        value_fn=lambda x: x.pollutants.pm25.concentration.value,
+    ),
+    AirQualitySensorEntityDescription(
+        key="so2",
+        translation_key="sulphur_dioxide",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
+        value_fn=lambda x: x.pollutants.so2.concentration.value,
+    ),
```

Comment:
device classes!

Reply:
Units of measurements don't match to the device class. There on case, where I could convert in the library from `ppb` to `ppm`.
But the others need conversion from `ppb` to micrograms per cubic meter, what we can't do.